docs/4_1_co-simulation_math.adoc

```diff
@@ -263,3 +263,14 @@ _In reality, every variable output argument is computed by one_ `fmi3Get{Variabl
 
 _In *Step Mode* the input arguments to_ latexmath:[\mathbf{f}_{\mathit{doStep}}] _are defined by calls to_ `fmi3Set{VariableType}` _functions._
 _The variables computed by_ latexmath:[\mathbf{f}_{\mathit{doStep}}] _can be inquired by_  `fmi3Get{VariableType}` _function calls.]_
+
+==== Smoothness, Continuity and Discontinuity
+
+Since values of variables will be set at specific communication points by the importing tool, the FMU must make assumptions about the values between these communication points.
```

Comment:
"Since values of variables will be set ..." this is true only for inputs, right? So shouldn't we talk about inputs instead of variables her?

Reply:
Agreed, fixed in next commit (I think).